spec/rails_helper.rb

```diff
@@ -1,5 +1,8 @@
 require "simplecov"
-SimpleCov.start
+SimpleCov.start do 
+  add_filter 'test_helper.rb'
+  add_filter 'application_record.rb'
+end
```

Comment:
This looks great, thanks for adding this and I am glad that we figured out the SimpleCov issue with the coverage percentage.